src/components/BatchModal.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <ion-header>
+    <ion-toolbar>
+      <ion-buttons slot="start">
+        <ion-button @click="closeModal">
+          <ion-icon :icon="closeOutline" />
+        </ion-button>
+      </ion-buttons>
+      <ion-title>Batch 2</ion-title>
+    </ion-toolbar>
+  </ion-header>
+
+  <ion-content>
+    <ion-item>
+      <ion-label position="fixed">Batch name</ion-label>
+      <ion-input placeholder="Batch 2" />
+    </ion-item>
+    <ion-item>
+      <ion-label position="fixed">{{ $t("Schedule") }}</ion-label>
+      <ion-input type="date" />
```

Comment:
I think we can use Ionic's new date picker component here.

Reply:
Ok sir